sdk-cpp/include/do_exceptions.h

```diff
@@ -1,21 +1,28 @@
-#pragma once
+#ifndef _DELIVERY_OPTIMIZATION_DO_EXCEPTIONS_H
+#define _DELIVERY_OPTIMIZATION_DO_EXCEPTIONS_H
 
 #include <exception>
 #include <stdint.h>
 #include <system_error>
 
+#if (DO_INTERFACE_ID == DO_INTERFACE_ID_COM)
+#include <comdef.h>
```

Comment:
>comdef [](http://example.com/codeflow?start=10&length=6)

Why is comdef.h needed here?

Reply:
FAILED() in line 58, though I think we can replace with winerror.h or by just defining the same macro inline:

#define FAILED(Status) ((HRESULT)(Status)<0)

I'll replace with winerror.h

Reply:
Use <> for 3rd party headers.